scripts/setup-webview-test-env.js

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env node
+
+const {exec} = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+const util = require('util');
+
+const execPromise = util.promisify(exec);
+
+async function main() {
+	// Empty print line to pretty-ify command line output
+	console.log();
+
+	// Copy webview test environment locally if it does not already exist
+	if (!fs.existsSync('./test-webview')) {
+		try {
+			console.log(color(['dim'], 'Copying webview test environment locally...'));
+			await execShellCommand('npx degit microsoft/vscode-webview-ui-toolkit-samples/all-components test-webview');
+		} catch (err) {
+			console.log(`${color(['red'], 'Error: Could not copy webview test environment locally')}\n    ${err}`);
+			process.exit();
+		}
+	}
+
+	// Install the webview test environment dependencies if they do not exist
+	if (!fs.existsSync('./test-webview/node_modules')) {
+		try {
+			console.log(color(['dim'], 'Installing webview test environment dependencies...'));
+			await execShellCommand('cd ./test-webview && npm install');
+		} catch (err) {
+			console.log(`${color(['red'], 'Error: Could not install webview test environment dependencies')}\n    ${err}`);
+			process.exit();
+		}
+	}
+
+	// Copy latest toolkit build into the webview test environment
+	console.log(color(['dim'], 'Copying latest toolkit build into webview test environment...'));
+	delDir('./test-webview/node_modules/@vscode/webview-ui-toolkit/dist');
+	createDir('./test-webview/node_modules/@vscode/webview-ui-toolkit/dist');
+	copyDir('./dist', './test-webview/node_modules/@vscode/webview-ui-toolkit');
+
+	// Print success and next steps messages
+	console.log();
+	console.log(color(['bold', 'green'], 'Webview test environment successfully configured!'));
+	console.log();
+	console.log('Next steps:');
+	console.log(`  1. Open the ${color(['cyan'], 'test-webview')} folder in a new VS Code window`);
+	console.log(`  2. Press ${color(['cyan'], 'F5')} to open the webview test extension with the most recent toolkit build loaded`);
+	console.log(`  3. Run the "${color(['cyan'], 'Webview UI Toolkit: All Components')}" command using the VS Code command palette`);
```

Comment:
For later: I wonder if this should be something like "Open All Components gallery" or similar

Reply:
Yeah we could! We would just have to update the all-components sample extension. But that's not hard to do at all :)

Reply:
Yeah, I may be wrong but I remember thinking we could rename those command contribs to use verbs. No rush but something to think about when we have a free min :)

Reply:
Yeah I think that would be a great idea actually!